tower-balance/src/lib.rs

```diff
@@ -1,10 +1,10 @@
 //! Load balancing middlewares.
 
-#![doc(html_root_url = "https://docs.rs/tower-balance/0.1.0")]
+#![doc(html_root_url = "https://docs.rs/tower-balance/0.3.0-alpha.1")]
 #![deny(missing_docs)]
 #![deny(rust_2018_idioms)]
 #![allow(elided_lifetimes_in_paths)]
-#![deny(warnings)]
+//#![deny(warnings)]
```

Comment:
?

Reply:
Doing large refactors when warnings also cause errors makes the process a pain. Just failed to change it back :sweat_smile:

Reply:
Separately, I _really_ dislike this `deny(warnings)` business, since it causes semver-compatible changes of dependencies to be breaking changes for us. For example, `IndexMap` has deprecated some methods, and that now causes `tower-balance` to no longer compile, even though our `Cargo.toml` hasn't changed.